types/ref-napi/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Same here.

Reply:
@uniqueiniquity This was included as part of this PR #21009
which was introduced because of this #20820 (comment)